docs/resources/cluster.md

```diff
@@ -311,6 +312,60 @@ Optional:
 - `time_interval_in_days` (Number) The time interval in days for the backup schedule.
 
 
+<a id="nestedatt--cmk_spec"></a>
+### Nested Schema for `cmk_spec`
```

Comment:
@shreyas-agnihotri @NValsan  - is CMK the external name of the feature? Do we need a better string here?

Reply:
Yeah, we are calling it CMK in docs. https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-secure-clusters/managed-ear/